azurerm/internal/services/loganalytics/tests/log_analytics_workspace_resource_test.go

```diff
@@ -307,3 +326,29 @@ resource "azurerm_log_analytics_workspace" "test" {
 }
 `, data.RandomInteger, data.Locations.Primary, data.RandomInteger)
 }
+
+func testAccAzureRMLogAnalyticsWorkspace_withVolumeCap(data acceptance.TestData, volumeCapGb float64) string {
+	return fmt.Sprintf(`
+provider "azurerm" {
+  features {}
+}
+	
+resource "azurerm_resource_group" "test" {
+  name     = "acctestRG-%d"
+  location = "%s"
+}
+
+resource "azurerm_log_analytics_workspace" "test" {
+  name                = "acctestLAW-%d"
+  location            = azurerm_resource_group.test.location
+  resource_group_name = azurerm_resource_group.test.name
+  sku                 = "PerGB2018"
+  retention_in_days   = 30
+  daily_quota_gb      = %d
```

Comment:
Incorrect type here for float:
```suggestion
  daily_quota_gb      = %f
```